lib/wallaby/experimental/selenium/webdriver_client.ex

```diff
@@ -336,6 +336,17 @@ defmodule Wallaby.Experimental.Selenium.WebdriverClient do
       do: {:ok, value}
   end
 
+  @doc """
+  Executes javascript asynchronously, taking as arguments the script to execute,
```

Comment:
I looked into it and I don't think it executes the script `asynchronously`, when executing the script the WebDriver client will block until the callback is called. Maybe we could reword this?